fibonacci.py

```diff
@@ -0,0 +1,22 @@
+# Automation onboarding
+# Fibonacci series: 0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144…
+
+def fibonacci(maxN):
```

Comment:
Review this variable name conventions on Python here :) https://visualgit.readthedocs.io/en/latest/pages/naming_convention.html#instance-variables 

it should be something like: `max_number`, review other variables in this PR

Reply:
Thanks @ovalerio280788 
I have changed the instant variable names. Please help take a look :)